.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish package to NPM
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust:
+          - stable
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup Node.js environment
```

Comment:
Do you need names for steps like actions/checkout, setup-x, or npm install? They are already clear imo.